frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -28,7 +28,8 @@ import {
 
 import { IconSize } from '@/enum/IconEnum';
 import { Severity } from '@/enum/SeverityEnum';
-import { pushNotification } from '@/store/NotificationState';
+import { pushNotification, setNotificationMsg } from '@/store/NotificationState';
```

Comment:
if we don't call pushNotification in this file, let's remove the import

Reply:
sure, I'll remove it